src/dbmanager.cpp

```diff
@@ -911,13 +911,7 @@ QList<NodeData> DBManager::readOldNBK(const QString fileName)
         QFile file(fileName);
         file.open(QIODevice::ReadOnly);
         QDataStream in(&file);
-#if QT_VERSION >= QT_VERSION_CHECK(5, 6, 0)
-        in.setVersion(QDataStream::Qt_5_6);
-#elif QT_VERSION >= QT_VERSION_CHECK(5, 4, 0)
-        in.setVersion(QDataStream::Qt_5_4);
-#elif QT_VERSION >= QT_VERSION_CHECK(5, 2, 0)
-        in.setVersion(QDataStream::Qt_5_2);
-#endif
+        in.setVersion(QDataStream::Qt_5_9);
```

Comment:
@nuttyartist Do you have any old file(s) I can use for testing? And do you know which version would be best for this?

Reply:
I think most people probably use a QT version >= 5.6, so the `QT_VERSION_CHECK` would make them use `QDataStream::Qt_5_6`, so that's what I set it to for now.